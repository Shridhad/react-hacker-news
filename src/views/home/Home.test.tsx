import React from 'react';
import ReactDOM from 'react-dom';
import HomeView from './home';

describe("<HomeView />", () => {
    it('renders HomeView', () => {
        const div = document.createElement('div');
        ReactDOM.render(<HomeView />, div);
        
        expect(div).toMatchSnapshot();
        ReactDOM.unmountComponentAtNode(div);
    });
});
