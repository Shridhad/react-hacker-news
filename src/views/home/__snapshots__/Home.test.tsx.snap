// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeView /> renders HomeView 1`] = `
<div>
  <div
    class="home-view"
  />
</div>
`;
